### PR TITLE
POLCT-1035: Add support for CodeDx placeholders

### PR DESCRIPTION
**Pull request by sig-chakrava**
As a part of supporting CodeDx, we will be supporting optional CodeDx section in io-manifest file, so we may need to resolve CodeDx placeholders if any specified.

Reference -
https://sig-confluence.internal.synopsys.com/display/IO/IO-CodeDx+Integration+-+2021.08+Release#IOCodeDxIntegration2021.08Release-Workflow

### CODE REVIEW

File: prescription.sh [hunk 1]
```diff
@@ -96,6 +96,10 @@ function generateYML () {
         --coverity.stream=*) coverity_stream="${i#*=}" ;;
         --coverity.username=*) coverity_username="${i#*=}" ;;
         --coverity.password=*) coverity_password="${i#*=}" ;;
+        --codedx.url=*) codedx_server_url="${i#*=}" ;;				#codedx
```
Comment (sig-suhaslr) [inline prescription.sh:99]: In manifest file the codedx url is referenced as `<<CODEDX_INSTANCE_URL>>` Can we change `codedx_server_url` to `codedx_instance_url` to keep the variables name consistent with the name in manifest file ?

Comment (sig-chakrava) [reply prescription.sh:99]: Changed `CODEDX_INSTANCE_URL` to `CODEDX_SERVER_URL` now to keep it consistent with other tools url related placeholders

Comment (sig-suhaslr) [reply prescription.sh:99]: Thanks

**Review by sig-suhaslr — APPROVED** [review]
LGTM

**Review by rohsa — APPROVED** [review]

**Review by sig-saraf — APPROVED** [review]

**Pull request merged**